src/igvfd/schemas/file.json

```diff
@@ -77,7 +77,11 @@
             "description": "The type of content in the file.",
             "comment": "This property is overwritten by the subclasses to define specific enum values.",
             "type": "string",
-            "enum": []
+            "enum": [],
+            "submissionExample": {
+                "appscript": "alignments",
+                "igvf_utils": "alignments"
+            }
         },
         "dbxrefs": {
```

Comment:
Missing example here I think

Reply:
oh, i think i didn't put an example here because there is no regex for it yet. We don't know if we will do SRA or other databases. @iyoungworth

Reply:
This makes sense to me, I think it's fine to leave it without an example